[IMP] website: add tips and new features to SEO page (#15245)
odoo/documentation: content/applications/websites/website/structure/seo.rst

speed.

.. note::

>    - Several modules are provided to help build the website’s content, such as

Reviewer: ```suggestion
   - Several modules are provided to help build the website's content, such as
```

   :alt: Optimize SEO

.. note::
   Changing the title of a blog post or the name of a product automatically updates the link to the
   related webpage. The old link still functions as a
   :ref:`301 redirect <website/pages/URL-redirection>` is created, maintaining the SEO.

.. tip::
   Click :guilabel:`Fill with AI` to automatically generate a meta title, a description, and benefit

>    from suggested keywords with a single click.

Reviewer: ```suggestion
   Click :guilabel:`Fill with AI` to automatically generate a meta title, a description, and get
   keyword suggestions.
```

>    Remove incorrect URL's from the :guilabel:`Broken Link` field and select valid ones to

Reviewer: ```suggestion
   Remove incorrect URLs from the :guilabel:`Broken Link` field and select valid ones to
```

guarantee that rankings will remain unchanged, follow these best practices to significantly reduce
the risk.

- Keep the existing content.

- Implement :ref:`redirects <website/pages/URL-redirection>` from old URLs to their new counterparts.

- Monitor traffic and indexation to make sure that everything is going well, using
  :doc:`../configuration/google_search_console`.

> By doing this, search engines reindex the site and maintain the visibility in search results.

Reviewer: ```suggestion
By doing this, search engines reindex the site and maintain its visibility in search results.
```

.. seealso::
   :doc:`../configuration/translate`

Page translation
----------------

You can translate an entire page in one click by going to the Website page, switching to the language
that requires a translation, and on the top-right corner, click on :menuselection:`Edit --> Translate`.
The text that is not translated appears in yellow. Click on the green button at the right to translate
the text.

Google merchant Center
----------------------

The Google Merchant Center is useful to publish your listings on other platforms (Instagram, facebook
Tiktok) by generating an XML feed and automatically synchronizes all the changes made in Odoo
directly in the new platform.

Go to :menuselection:`Website --> Configuration --> Settings` tick the Google Merchant Center box,
click on the :guilabel:`Copy URL` button. Then go to the Google Merchant website, paste the copied

> link and click on Save.

Reviewer 2: Larissa recently wrote a whole page about [Google Merchant Center](https://www.odoo.com/documentation/19.0/applications/websites/ecommerce/google_merchant_center.html). This section doesn’t appear to be related to SEO, so it may be out of scope for this change. I’d strongly recommend removing this paragraph.
